liveblog/liveblog
# [LBSD-2852] - Error switching from Default to Simple Theme

### Purpose

This PR fixes a bug that was causing an issue when changing from the Default theme to the Simple theme.

### What has changed

1. Added `semver` package to help in themes version updating. These also allows themes to have prerelease tags which should correspond with the system child themes.
2. Updated Simple and AMP theme to point to the new prerelease default theme.

### Steps to test
1. Open the Editor.
2. Create a new blog and choose Default theme as the theme to start with. 
3. Add several posts. Click Live Preview to see the blog in the default theme.
4. Go to the blog settings and change the theme to Simple theme. Click Save. Go back to the Editor and open Live Preview to see the blog in the simple theme.
5. Go to the blog settings and change the theme to AMP theme. Click Save. Go back to the Editor and open Live Preview to see the blog in the AMP theme.
6. Observe that it works with no errors.


Resolves: [LBSD-2852](https://sofab.atlassian.net/browse/LBSD-2852)

[LBSD-2852]: https://sofab.atlassian.net/browse/LBSD-2852?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
